### PR TITLE
Riak client upgrade

### PR DESCRIPTION
**Pull request by SavithaK**
- Upgraded riak client version to latest (2.3.2)
- Removed all references to http for riak
According to https://github.com/basho/riak-ruby-client/blob/master/RELEASE_NOTES.md:
HTTP support has been removed from the Riak Ruby Client in favor of focusing on Protocol Buffers.
- For some reason my spec failed with uninitialized constant Set, hence requiring set.
- Reinstalling riak seems to keep it running. Credits due to https://github.com/mostlyserious/riak-js/pull/225/files

PR for https://github.com/braintree/curator/issues/53

@pgr0ss @bmorton @chrismohr

### CODE REVIEW

**Comment by SavithaK** [comment]
@pgr0ss tests failed because of they couldn't connect to pb port 8087. Is this a travis config?

**Comment by pgr0ss** [comment]
I've had a lot of problems with riak on travis. You could try adding an `/etc/default/riak` in a `before_install`:

https://github.com/travis-ci/travis-ci/issues/3328
https://docs.travis-ci.com/user/installing-dependencies/

**Comment by SavithaK** [comment]
Don't seem to be having luck with before install :(

**Comment by SavithaK** [comment]
@pgr0ss I finally got some time to look into it and get it working. Mind if we stop supporting 1.9.3 from now on? I am not sure why installing it fails, but I am guessing its old enough that we can safely stop supporting it?

**Comment by SavithaK** [comment]
@pgr0ss if you're good with this please merge it.

**Comment by SavithaK** [comment]
Hullo @pgr0ss! Any thoughts on merging this in?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 require 'timecop'
 
 require 'ammeter/init'
+require 'set'
```
Comment (pgr0ss) [inline spec/spec_helper.rb]: What is this for?

Comment (pgr0ss) [inline spec/spec_helper.rb]: Oh, I see your note. Strange.

**Comment by pgr0ss** [comment]
Sorry for the slow reply. Thanks for upgrading the client!

**Comment by SavithaK** [comment]
Thanks @pgr0ss :)

**Comment by SavithaK** [comment]
@pgr0ss Also when you get a chance, could you bump the version?

**Comment by pgr0ss** [comment]
Done.

**Pull request merged**